src/algebra/group_ring_action.lean

```diff
@@ -40,7 +40,11 @@ class mul_semiring_action (M : Type u) (R : Type v) [monoid M] [semiring R]
 
 section semiring
 
-variables (M G : Type u) [monoid M] [group G]
+<<<<<<< HEAD
+variables (M G : Type*) [monoid M] [group G]
+=======
+variables (M N G : Type*) [monoid M] [monoid N] [group G]
+>>>>>>> 52593b3d999f9b0d76af36baf659ec1638110f35
```

Comment:
This is a syntax error, it looks like you created local merge conflicts

Reply:
OK, I changed the variables. I am suspecting that I can only make `N` implicit but not `M` because `M` is used in other places and for some reason I get errors.